Assets/Editor Default Resources/Previews/MeshPreviewShader.shader

```diff
@@ -39,6 +39,10 @@ Shader "Unlit/Mesh-MultiPreview"
                 float4 uv1 : TEXCOORD1;
                 float4 uv2 : TEXCOORD2;
                 float4 uv3 : TEXCOORD3;
+                float2 uv4 : TEXCOORD4;
```

Comment:
Why some of them are float2s while others are float4s?

Reply:
:oof: